### PR TITLE
:sparkles: Add CAPX Nutanix Provider

### PR DESCRIPTION
**Pull request by tuxtof**
**What this PR does / why we need it**:

Adding the cluster-api-provider-nutanix provider - [cluster-api-provider-nutanix](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix)

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

no issues fixed in this PR

### CODE REVIEW

File: docs/book/src/user/quick-start.md [hunk 1]
```diff
@@ -329,6 +329,11 @@ Please visit the [Hetzner project][Hetzner provider].
 
 Please visit the [Metal3 project][Metal3 provider].
 
+{{#/tab }}
+{{#tab Nutanix}}
+
+Please follow the Cluster API Provider for [Nutanix Getting Started Guide](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/blob/main/docs/getting_started.md)
```
Comment (killianmuldoon) [inline docs/book/src/user/quick-start.md:335]: nit

The guide at this link should be updated to reference 0.2.2 or 0.2.1. It looks like 0.2.0 doesn't exist so the commands don't work.

File: docs/book/src/reference/glossary.md [hunk 2]
```diff
@@ -56,6 +56,9 @@ Cluster API Provider IBM Cloud
 ### CAPN
 Cluster API Provider Nested
 
+### CAPX
```
Comment (killianmuldoon) [inline docs/book/src/reference/glossary.md:59]: Cool acronym! 

We're going to have to make sure to refer all providers as CAP* from now on :slightly_smiling_face:

**Review by killianmuldoon — COMMENTED** [review]
Looking good!

Can you also add Nutanix as part of the tab in "Deploying a CNI solution" in the Cluster API book quick start guide. Unless there's some reason to exclude it? (I have 0 knowledge of how nutanix works)

File: docs/book/src/reference/glossary.md (hunk 2 above)
Comment (tuxtof) [reply docs/book/src/reference/glossary.md:59]: yes i like it too

X is clearly our Nutanix brand ;-)

File: docs/book/src/user/quick-start.md (hunk 1 above)
Comment (tuxtof) [reply docs/book/src/user/quick-start.md:335]: let me fix it , doc is not yet dynamic and i was so impatient to push this PR :-D

Comment (tuxtof) [reply docs/book/src/user/quick-start.md:335]: doc ix fixed thanks @killianmuldoon

**Comment by sbueringer** [comment]
Looks good. @tuxtof can you please merge the commits?

(we have to use Prow in merge-mode for release note generation, and that mode preserves all commits when merging)

**Comment by tuxtof** [comment]
ok @sbueringer commit are squashed

**Comment by sbueringer** [comment]
/assign @fabriziopandini 

@tuxtof Should we cherry-pick so it's available in the next v1.1.x release of CAPI/clusterctl?

**Comment by tuxtof** [comment]
If possible yes please 😎

**Comment by sbueringer** [comment]
/cherry-pick release-1.1

No problem. I think it's reasonable to support new providers in our latest release series so they don't have to wait until the next minor release.

**Comment by fabriziopandini** [comment]
/approve cancel
for CI to complete

**Comment by sbueringer** [comment]
@tuxtof Looks like there were some merge conflict when the bot tried to create the cherry-pick. Can you please open the ~ same PR against release-1.1 (manually)? Thank you!!

**Comment by tuxtof** [comment]
@sbueringer yes of course let me do that 
when the next release is planned ?

**Comment by tuxtof** [comment]
Hello @sbueringer 
PR for branch release-1.1 is here #6378

**Comment by sbueringer** [comment]
> @sbueringer yes of course let me do that when the next release is planned ?

Not sure, usually patch releases are ~ monthly. (cc @fabriziopandini )

**Pull request merged**